### PR TITLE
Vue setup

### PR DESCRIPTION
**Pull request by adw45**
To run locally:
```bash
npm install -g webpack
webpack
npm start
```

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "presets": ["es2015", "stage-0"],
+    "plugins": ["transform-runtime"]
+}
```
Comment (jafern14) [inline .babelrc]: nit poop

File: app/js/room.js [hunk 2]
```diff
@@ -10,15 +10,15 @@ function getParameterByName(name, url) {
```
Comment (adw45) [inline app/js/room.js]: this file can go away.

**Comment by adw45** [comment]
Fix `npm start` to inclue webpack

File: app/components/start-room.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    template: `
+        <div>
+            <h1>{{ title }}</h1>    
+            <input type="text" id="roomName" placeholder="Room name" />
+            <button @click="joinRoom()">Join Room</button>
+        </div>
+    `,
+    methods: {
+        joinRoom: function() {
+            this.$router.push( {name: 'room', params: { id: jQuery("#roomName").val() } } )
```
Comment (jafern14) [inline app/components/start-room.js]: remove jquery entirely 💩

Comment (adw45) [inline app/components/start-room.js]: Fuck dat

Comment (jafern14) [inline app/components/start-room.js]: l2vue

**Comment by jafern14** [comment]
Who approved this pull request? I sure didn't.

**Comment by jafern14** [comment]
This wasn't ready for merge. Please revert merged PR and open this back up for more edits.

**Comment by jafern14** [comment]
Shit ass dev @adw45

**Pull request merged**